arch/Kconfig

```diff
@@ -496,6 +498,8 @@ config IRQ_OFFLOAD_NESTED
 
 config EXTRA_EXCEPTION_INFO
 	bool "Collect extra exception info"
+	# GDB on ARM needs the etxra registers
+	default y if GDBSTUB && ARM
```

Comment:
I assume those extra (type: etxra) registers are mandatory, this should be done via `select` (under `arch/arm/core/aarch32/Kconfig` as mentioned above). Using `default` here means that it can be overridden via command line or defconfig downstream of arch.